dashboard/db/schema.rb

```diff
@@ -425,6 +425,7 @@
     t.datetime "created_at",                   null: false
     t.datetime "updated_at",                   null: false
     t.integer  "script_id",        limit: 4
+    t.boolean  "started"
```

Comment:
should this have `default: false, null: false` or is there value in allowing null?